modules/op-speccancel-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,35 @@
+// This module is included in the following assemblies:
+// * pac/managing-pipeline-runs-pac.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="speccancel-pipeline-run-using-pipelines-as-code_{context}"]
+= Annotations for specifying automatic cancellation-in-progress for a pipeline run
+
+By default, {pac} does not cancel pipeline runs automatically. Every pipeline run that {pac} creates and starts executes until it completes. However, events that trigger pipeline runs can come in quick succession. For example, if a pull request triggers a pipeline run and then the user pushes new commits into the pull request source branch, each push triggers a new copy of the pipeline run. If several pushes happen, several copies can run, resulting in a drain on cluster resources.
```

Comment:
This is a correct use of the term _drain_, I think. However, in the OCP docs, it seems that _drain_ is almost always (if not always) in reference to the MCO draining pods from a node. Maybe use a different word here?

```suggestion
By default, {pac} does not cancel pipeline runs automatically. Every pipeline run that {pac} creates and starts executes until it completes. However, events that trigger pipeline runs can come in quick succession. For example, if a pull request triggers a pipeline run and then the user pushes new commits into the pull request source branch, each push triggers a new copy of the pipeline run. If several pushes happen, several copies can run, which can consume excessive cluster resources.
```